__tests__/app.test.js

```diff
@@ -168,4 +167,94 @@ describe("api/articles", () => {
         });
     });
   });
+  describe("POST /api/articles/:article_id/comments", () => {
+    test("Status 201: creates new comment and responds with the posted comment", () => {
+      return request(app)
+        .post("/api/articles/9/comments")
+        .send({
+          username: "rogersop",
+          body: "I love articles like this.",
+        })
+        .expect(201)
+        .then(({ body }) => {
+          expect(body).toMatchObject({
+            body: "I love articles like this.",
+            votes: 0,
+            author: "rogersop",
+            article_id: 9,
+            comment_id: 19,
+          });
+          expect(body).toHaveProperty("created_at", expect.any(String));
```

Comment:
As a tip, you could include the created_at key in your toMatchObject above and set the value to expect.any(String) and it will work the same way :)